tests/drivers/disk/disk_access/testcase.yaml

```diff
@@ -8,9 +8,10 @@ tests:
     filter: CONFIG_SDHC
     extra_configs:
       - CONFIG_DISK_DRIVER_SDMMC=y
+      - platform:mimxrt1060_evk@C/mimxrt1062/qspi:CONFIG_MAIN_STACK_SIZE=40960
     tags: mcux
     integration_platforms:
-      - mimxrt1060_evk/mimxrt1062/qspi
+      - mimxrt1060_evk@C/mimxrt1062/qspi
```

Comment:
given that rev C is the default, I'm not sure we have to explicitly call it out here because twister will generate an alias list of [mimxrt1060_evk/mimxrt1062/qspi, mimxrt1060_evk@C/mimxrt1062/qspi]

Reply:
@dleach02, I would prefer to have a clear one, as I have multiply boards, which would be very confuse to configure them in board farm

Reply:
fair enough. I actually wasn't sure how best to do this as there are a bunch of testcase/sample yaml files that I touched.

Reply:
CI will tracking these, to make sure all items within test plan, will be maintained.